src/components/Match/renderMatch.js

```diff
@@ -266,13 +266,15 @@ function renderMatch(m) {
   });
 
   if (m.objectives && m.objectives.length > 0) {
-    m.objectives.map(obj => (
-      obj.player_slot === undefined && (
-        obj.slot > 4
-          ? Object.defineProperty(obj, 'player_slot', { value: obj.slot + 123 })
-          : obj.slot > 0 && Object.defineProperty(obj, 'player_slot', { value: obj.slot })
-      )
-    ));
+    m.objectives.forEach((obj) => {
+      if (obj.slot >= 0 && obj.player_slot === undefined) {
+        if (obj.slot > 4) {
+          Object.defineProperty(obj, 'player_slot', { value: obj.slot + 123 });
```

Comment:
> is the purpose of Object.defineProperty to avoid the eslint warning about assignment to function parameter?

yes, and I think the current solution is the most proper here since no need to define player_slot if slot = -1 that means creeps I guess 🦄